### ISSUE TITLE
Minified version cannot dereference correctly

### ISSUE THREAD

**Issue by egekorkan** [digitalbazaar/jsonld.js#268]
I have the following json-ld document: [https://github.com/w3c/wot/blob/master/plugfest/2018-sept-online/TDs/Siemens/FestoLive-Oracle.jsonld](https://github.com/w3c/wot/blob/master/plugfest/2018-sept-online/TDs/Siemens/FestoLive-Oracle.jsonld)

With node.js v10.11.0, Firefox browser v 62.0.2, jsonld.js v 1.1.0 if I use the `expand` or `toRDF` methods on this document I get an error (listed in the end). 
I am using the minified version of this package. I have used the one provided by the package in the `node_modules/dist/jsonld.min.js` and I have also minified the normal package with uglify-js. I have the exact same error with both minified packages.
However, if I use the normal package, I don't have any errors. JSON-LD Development Playground validates it as well.

```
 { jsonld.InvalidUrl: Dereferencing a URL did not result in a valid JSON-LD object. Possible causes are an inaccessible URL perhaps due to a same-origin policy (ensure the server uses CORS if you are using client-side JavaScript), too many redirects, a non-JSON response, or more than one HTTP Link Header was provided for a remote context.
    at new JsonLdError (./myDir/jsonld.min.js:1:15293)
    at _callee2$ (./myDir/jsonld.min.js:1:21579)
    at tryCatch (./myDir/jsonld.min.js:1:144271)
    at Generator.invoke [as _invoke] (./myDir/jsonld.min.js:1:148075)
    at Generator.prototype.(anonymous function) [as throw] (./myDir/jsonld.min.js:1:145395)
    at step (./myDir/jsonld.min.js:1:23706)
    at ./myDir/jsonld.min.js:1:23886
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
    at startup (internal/bootstrap/node.js:279:19)
  name: 'jsonld.InvalidUrl',
  details:
   { code: 'loading remote context failed',
     url: 'http://www.w3.org/ns/td',
     cause:
      { jsonld.LoadDocumentError: URL could not be dereferenced, an error occurred.
          at new JsonLdError (./myDir/jsonld.min.js:1:15293)
          at _callee$ (./myDir/jsonld.min.js:1:313597)
          at tryCatch (./myDir/jsonld.min.js:1:144271)
          at Generator.invoke [as _invoke] (./myDir/jsonld.min.js:1:148075)
          at Generator.prototype.(anonymous function) [as throw] (./myDir/jsonld.min.js:1:145395)
          at step (./myDir/jsonld.min.js:1:311947)
          at ./myDir/jsonld.min.js:1:312127
          at process._tickCallback (internal/process/next_tick.js:68:7)
          at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
          at startup (internal/bootstrap/node.js:279:19) name: 'jsonld.LoadDocumentError', details: [Object] } } }

```

**Comment by egekorkan** [comment]
@mattcollier I have used an example from the upcoming TD standard where they had agreed on using http. It is seen here: https://w3c.github.io/wot-thing-description/#

By the way, the json ld document is stored on my computer.

**Comment by dlongley** [comment]
@egekorkan,

This looks to be some kind of CORS issue, though I don't yet know why the minified version would behave differently from the non-minified version. Perhaps there is a polyfill that is being injected that changes which headers are sent. It looks like the playground is using a proxy service to retrieve the context, so we can't really rely on that to tell us if the CORS stuff is kosher.

Can you paste the headers that are being sent when the request is made? Perhaps there's a "non-simple" header in there that is causing the response to be rejected. I don't see an `Access-Control-Allow-Headers` header in the response when I curl the context.

**Comment by egekorkan** [comment]
So I have used Wireshark to see the header information. Actually, when I run with the minified version, there is no data that is being sent. I think that the dereferencing method/engine is causing the problem even before sending something.
P.S. I do see the request on Wireshark when I don't use the minified version but the normal package.

**Comment by davidlehn** [comment]
I wrote a quick test case that works in recent chromium and firefox.  Is this similar to what you are doing? 
 If not, could you provide a test case we could run?  Is your browser network dev tool showing the traffic?  Curious why you had to dig down to wireshark level for this.  If you're serving your page over https, there could be mixed security issue when jsonld tries to load non-https http://www.w3.org/ns/td.  I think we've worked around such things in the past with a custom document loader that rewrites w3.org urls to always be https.  That may not be the issue here though.
```html
<!DOCTYPE html>
<html>
<head>
  <script src="https://unpkg.com/jsonld@1.1.0/dist/jsonld.min.js"></script>
  <script>
    let url = 'https://raw.githubusercontent.com/w3c/wot/master/plugfest/2018-sept-online/TDs/Siemens/FestoLive-Oracle.jsonld';
    jsonld.expand(url)
      .then(result => console.log('RESULT', result))
      .catch(err => console.error('ERROR', err));
  </script>
</head>
<body></body>
</html>
```

**Comment by egekorkan** [comment]
I have dumped my code here. I was using node.js.  
http://ronsoros.github.io/?afb0673155aecc52484b1df8d93ad1c7259a7a99

**Comment by egekorkan** [comment]
This problem was fixed by using the non minified version but Safari browser has the same problem even with the non minified version

**Comment by dlongley** [comment]
Closing as either fixed or now a duplicate of #399.
